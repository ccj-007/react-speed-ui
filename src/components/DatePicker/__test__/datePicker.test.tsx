import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { DatePicker } from '../../index';

describe('test DatePicker component', () => {
  it('should render', () => {
    const { asFragment } = render(<DatePicker />);
    expect(asFragment()).toMatchSnapshot();
  });
});
