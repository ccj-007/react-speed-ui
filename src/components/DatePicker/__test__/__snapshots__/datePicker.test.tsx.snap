// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test DatePicker component should render 1`] = `
<DocumentFragment>
  <div
    style="display: flex;"
  >
    <div
      class="speed-datePickerItem"
      style="width: 200px; height: 36px; position: relative;"
    >
      <div
        class="speed-input-warp"
        style="width: 200px;"
      >
        <input
          class="speed-input"
          data-testid="input"
          placeholder="输入你的日期"
          readonly=""
          style="padding-right: 30px; width: 200px; height: 36px;"
          value=""
        />
        <div
          class="speed-input-icon"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-calendar-days fa-1x speed-icon"
            color="#aaa"
            data-icon="calendar-days"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M128 0c17.7 0 32 14.3 32 32V64H288V32c0-17.7 14.3-32 32-32s32 14.3 32 32V64h48c26.5 0 48 21.5 48 48v48H0V112C0 85.5 21.5 64 48 64H96V32c0-17.7 14.3-32 32-32zM0 192H448V464c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V192zm64 80v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V272c0-8.8-7.2-16-16-16H80c-8.8 0-16 7.2-16 16zm128 0v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V272c0-8.8-7.2-16-16-16H208c-8.8 0-16 7.2-16 16zm144-16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V272c0-8.8-7.2-16-16-16H336zM64 400v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H80c-8.8 0-16 7.2-16 16zm144-16c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H208zm112 16v32c0 8.8 7.2 16 16 16h32c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H336c-8.8 0-16 7.2-16 16z"
              fill="currentColor"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
